### PR TITLE
WIP: chore(falcosidekick): Values and config Update

### PR DESCRIPTION
**Pull request by Lowaiz**
**What type of PR is this?**

/kind feature

/kind chart-release

**Any specific area of the project related to this PR?**

/area falcosidekick-chart

**What this PR does / why we need it**:

PR in prevision of FalcoSidekick version bump to 2.27.0. 
Updating Chart for PRs: 
falcosecurity/falcosidekick#336
falcosecurity/falcosidekick#337
falcosecurity/falcosidekick#338
falcosecurity/falcosidekick#341
falcosecurity/falcosidekick#346
falcosecurity/falcosidekick#347
falcosecurity/falcosidekick#349
falcosecurity/falcosidekick#353
falcosecurity/falcosidekick#356
falcosecurity/falcosidekick#359
falcosecurity/falcosidekick#360
falcosecurity/falcosidekick#362
falcosecurity/falcosidekick#366

**Which issue(s) this PR fixes**:

None

**Special notes for your reviewer**:

Do not merge this now.

/do-not-merge

**Checklist**

- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] CHANGELOG.md updated

### CODE REVIEW

**Comment by Issif** [comment]
Thanks for your PR, as more changes may come before the next release of Sidekick, let's keep it opened and I will ask you to add in future

**Comment by Lowaiz** [comment]
@Issif 
Everything in the 2.27 Milestone of the sidekick has now its config here

File: falcosidekick/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,16 @@ numbering uses [semantic versioning](http://semver.org).
...
 
+## 0.5.7
+
+* Add `alertmanager.extralabels` and `alertmanager.extraannotations` to the values,
+as a list of comma separated labels and annotations to add to the AlertManager trigger.
+* Add `smtp.autmechanism` and associated fields to be able to use any SASL mechanism.
+* Add `yandex.datastreams` and associated fields.
+* Add `nodered` and associated fields.
+* Add `mqtt` and associated fields.
+* Update `elasticsearch.type` default value to `_doc`
```
Comment (Issif) [inline falcosidekick/CHANGELOG.md:16]: Impressive, I created the PR less than 1h ago :+1:

**Comment by Issif** [comment]
Following this PR https://github.com/falcosecurity/falcosidekick/pull/346 we need now to export the env var `NAMESPACE` in all Falcosidekick pods, can you add it following this snippet please:
```yaml
 - name: NAMESPACE
   valueFrom:
     fieldRef:
       fieldPath: metadata.namespace
```

**Comment by alacuku** [comment]
Hi @Lowaiz, for the falco chart we came up with a solution to keep updated the `values` table. It uses the [helm-docs](https://github.com/norwoodj/helm-docs) that combined with a makefile generates an updated `values` table from the comments in the `values.yaml` file. Here you can find the commit that introduced `helm-docs` in falco: https://github.com/falcosecurity/charts/commit/35830761658320af8a6fd5a0101af4b52e8aae00. 
It would be nice to implement the same solution for falcosidekick! Let me know what you think.

File: falcosidekick/CHANGELOG.md (hunk 1 above)
Comment (Lowaiz) [reply falcosidekick/CHANGELOG.md:16]: Hehe :smile:

**Comment by Lowaiz** [comment]
> It would be nice to implement the same solution for falcosidekick!

Could be great, I'll take a look at that tomorrow.

**Comment by Lowaiz** [comment]
@alacuku https://github.com/falcosecurity/charts/pull/384

**Comment by Issif** [comment]
I released the new versions and updated the charts. I co-authored you of course. Thanks.

**Pull request closed without merging**